lib/client.ts

```diff
@@ -117,9 +117,11 @@ export class SiweMessage {
 			this.nonce = (Math.random() + 1).toString(36).substring(4);
 		}
 
+		const chainField = `Chain ID: ` + this.chainId || "1";
```

Comment:
If it's mandatory I suppose we want to make to make it explicit for the user?

Reply:
agreed, besides if we let this like it is could we make `Chain ID: ${this.chainId || '1'}`?